pages/_lang/publish.vue

```diff
@@ -805,12 +805,217 @@
       </section>
     </Modal>
 
+    <!-- New Edge Modal -->
+
+    <Modal
+      title="Windows Package Options"
+      :button_name="$t('modal.done')"
+      ref="windowsPWAModal"
+      :showSubmitButton="true"
+      @modalSubmit="windowsOptionsModalSubmitted"
+      @cancel="windowsOptionsModalCancelled"
+      v-on:modalOpened="modalOpened()"
+      v-on:modalClosed="windowsModalClosed()"
+      v-if="windowsForm"
+    >
+      <div id="topLabelBox" slot="extraP">
+        <label id="topLabel">Customize your Windows Package below</label>
+        <p class="l-generator-error" v-if="androidPWAError">
+          <i class="fas fa-exclamation-circle"></i>
+          {{ $t(androidPWAError) }}
+        </p>
+      </div>
+
+      <section class="androidModalBody androidOptionsModalBody">
+        <form style="width: 100%">
+          <div class="row">
+            <div class="col-lg-6 col-md-12">
+              <div class="form-group">
+                <label for="windowsPackageIdInput">
```

Comment:
Can you make this a link to https://github.com/pwa-builder/pwabuilder-windows-chromium-docs/blob/master/find-publisher.md